doc/vast.1.md

```diff
@@ -59,31 +59,60 @@ The following key components exist:
   Accepts query expressions from users, asks **index** for hits, takes them to
   **archive** to extract candidates, and relays matching events to **sink**s.
 
+### Schematic
+```
+                +--------------------------------------------+
+                | Node                                       |
+                |                                            |
+  +--------+    |             +--------+                     |    +-------+
+  | source |    |         +--->archive <------+           +-------> sink  |
+  +-----bro+-------+      |   +--------<---+  v-----------++ |    +---json+
+                |  |      |                |  | exporter   | |
+                | +v------++           +------>------------+ |
+     ...        | |importer|           |   |     ...         |      ...
+                | +^------++           |   |                 |
+                |  |      |            |   +-->------------+ |
+  +--------+-------+      |            |      | exporter   | |
+  | source |    |         |   +--------v      ^-----------++ |    +-------+
+  +----pcap+    |         +---> index  <------+           +-------> sink  |
+                |             +--------+                     |    +--ascii+
+                |                                            |
+                |                                            |
+                +--------------------------------------------+
+```
+The above diagram illustrates the default configuration of a single node and
+the flow of messages between the components. The **importer**, **index**, and
+**archive** are singleton instances within the **node**. **Source**s are spawned
+on demand for each ingestion. **Sink**s and **exporter**s form pairs that are
```

Comment:
> [..] for each ingestion [XXX]

where XXX could be "process", "activity", or "procedure". If you have a better suggestion, let me know.

Reply:
I don't understand why ingestion can't stand alone. "process" would qualify it in an incorrect way, and "activity" and "procedure" sound strange.

Reply:
To me, "ingestion" alone sounds not complete, even though it's grammatically correct. Referring to "an ingestion" is what I have trouble with, because it's not clear to me what it encompasses. By making it a two-component term, I find it clearer to understand.

However, I agree that none of my suggestions actually fit very well.

Reply:
https://whatis.techtarget.com/definition/data-ingestion is the first hit I get via google.
I left the "data" out because it can be guessed from the context, but we could add it back in. Alternative: "data import"?

Reply:
I like "data import", let's go with that!